debezium-testing/debezium-testing-testcontainers/pom.xml

```diff
@@ -70,6 +70,11 @@
       <artifactId>postgresql</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mongodb</groupId>
+      <artifactId>mongodb-driver-sync</artifactId>
+      <scope>test</scope>
```

Comment:
For testing the test container infra only (no downstream dependency introduced in clients).